src/components/DisplayInput.js

```diff
@@ -84,20 +93,20 @@ const DisplayInput = React.createClass({
         </div>
 
         <Row>
```

Comment:
That change was made after discussion with @derek-boman.  It was originally in the same row and positioned to the right as you suggest.  I think the main problem is that it's position to the right doesn't work for mobile.